packages/shell-api/src/bulk.spec.ts

```diff
@@ -73,7 +72,7 @@ describe('Bulk API', () => {
           const bulkWriteResult = {
             ok: 1,
             nInserted: 1,
-            insertedIds: { 0: new ObjectId() },
+            insertedIds: { 0: new bson.ObjectId() },
```

Comment:
I was hoping to avoid source changes entirely, I hope these don’t conflict with the linter update :crossed_fingers: